examples/pytorch/hilander/PSS/test.sh

```diff
@@ -0,0 +1,9 @@
+python Smooth_AP/src/evaluate_model.py \
+--dataset Inaturalist \
+--bs 384 \
+--source_path ~/code/Smooth_AP/data/ --embed_dim 128 \
+--resume "INATURALIST_RESNET50_2021-9-2-4-21-21/checkpoint.pth.tar" \
```

Comment:
Users won't have the INATURALIST_RESNET50_2021-9-2-4-21-21 , maybe we can put a place holder and let user type in the exact path.